packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -485,12 +489,18 @@ class CreationParams {
   /// When null the platform's webview default is used for the User-Agent header.
   final String? userAgent;
 
+  ///This value is required by the platforms [iOS] and [Android] in order to know
+  ///if we should attach the [RefreshControl] or [SwiperLayout] to the native [WebView].
+  ///
+  ///The default value is false.
+  final bool? hasPullToRefresh;
```

Comment:
This should be called `enablePullToRefresh` to make it clear that it's about controlling the setting, not describing how it works.